src/zenml/pipelines/base_pipeline.py

```diff
@@ -553,9 +553,10 @@ def run(
             if runs:
```

Comment:
Can we maybe extract this functionality to display the dashboard URL into a separate method?